### DISCUSSION TITLE
Difference between Predict module and CausalEffects module

### DISCUSSION THREAD

**Discussion by jvicentem** [jakobrunge/tigramite#221] [Q&A] [answered]
Hi all,

Probably this is a very basic question, but I truly don't understand the difference: what is the difference between the Predict module and the CausalEffects module? 

From what I see, with both I can modify the input variables, calculate a couple of predictions and get the difference between them. However, I suspect there's something else (probably a "big something") that I'm missing.

As you can imagine, these are my very first steps on the causal effects field.

**Comment by jakobrunge** [comment, marked as answer]
The Predict class is more of a forecast module and is focused on time-lagged predictions. If you want to predict causal effects, ie, the effect of interventions, rather use the CausalEffects class. One could say that the Prediction class is a time-lagged, direct-effect, multi-intervention special case of causal effects. It also doesn't allow to obtain adjustment sets to account for confounding as explained in the CausalEffects tutorial...
